pkg/generators/forwarding/fluentd/generators_test.go

```diff
@@ -0,0 +1,235 @@
+package fluentd
```

Comment:
Please rewrite this test in ginkgo/gomega style as the whole package is using this style. It has the benefits that you can use the more advanced config parsing matcher `EqualTrimedLines`. It integrates the test in this package suite.

Reply:
Fixed